src/v2/Apps/Artwork/Components/OtherWorks/index.tsx

```diff
@@ -86,6 +96,13 @@ export const OtherWorks = track()(
               )
               return (
                 <Box key={`Grid-${index}`} data-test={contextModule}>
+                  {artistSeriesIsEnabled &&
+                    grid.__typename === "ArtistArtworkGrid" && (
```

Comment:
What's this line doing?

Reply:
This is what ensures that this rail shows up above (and only) the 'other works by this artist' grid. Those grids are dynamic (coming from MP), and the `__typename` is what tells you what kind of grid it is ('other works from the gallery', 'other works from the show', etc.)